pytensor/tensor/basic.py

```diff
@@ -760,7 +760,12 @@ def switch(cond, ift, iff):
     """if cond then ift else iff"""
 
 
-where = switch
+# where = switch
+def where(cond, ift, iff):
```

Comment:
They should have default?

```suggestion
def where(cond, ift=None, iff=None):
```

Also should we raise if one is None but not the two?

Reply:
oh yea mb those changes didnt get added. numpy also raises an error if only one arg (apart from condition) is passed so i'll add that here as well